app/AppComponents.scala

```diff
@@ -1,8 +1,9 @@
 import com.gu.atom.play.ReindexController
 import config.LogConfig
-import config.Config.{config, permissions}
+import config.Config.{config, permissions, dynamoDB, capiDynamoDB, capiLambdaClient}
```

Comment:
With two `dynamoDB` configuration properties, do they have different values?
(One of the imports is unused but I'm just curious.)

Reply:
One is for access to dynamo in the capi account. If I merge them then it's so much the better.